index.html

```diff
@@ -64,123 +65,47 @@ <h2>My Recent Works</h2>
-              <div class="stories-info-desc">
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-              </div>
-              <div class="stories-info-tools">
-                <ul>
-                  <li>css</li>
-                  <li>html</li>
-                  <li>bootstrap</li>
-                  <li>ruby</li>
-                </ul>
-              </div>
-              <div class="stories-info-button">
-                <a href="https://github.com/NduatiKagiri/my_portfolio" id="stories-info-button">See Project</a>
+
+          <div id="myModal" class="modal">
+
+            <!-- Modal content -->
+            <div class="modal-content">
+              <span class="close">&times;</span>
+              <div class="modal-info">
+                <div class="modal-top">
+                  <div class="project-title">
+
+                  </div>
+                  <div class="project-tools">
+                    <ul>
+                      <li>HTML/CSS</li>
+                      <li>Ruby on Rails</li>
+                      <li>Javascript</li>
+                    </ul>
+                  </div>
+                  <div class="project-img">
+                    <img src="assets/img/jscomp.jpeg" alt="">
+                  </div>
+                </div>
+                <div class="modal-bottom">
+                  <div class="project-desc">
+                    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into</p>
+                    <p>electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum</p>
+                  </div>
+                  <div class="project-links">
+                    <a href="https://nduatikagiri.co.ke" id="project-link-1">See Live &nbsp; &nbsp; &nbsp; <i class="fas fa-external-link" aria-hidden="true"></i></a>
+                    <a href="https://github.com/NduatiKagiri/my_portfolio" id="project-link-2">See Source &nbsp; &nbsp; &nbsp; <i class="fab fa-github" aria-hidden="true"></i></a>
+                  </div>
+                  <div class="project-buttons">
+                    <button type="button" name="button"><i class="fal fa-long-arrow-left" aria-hidden="true"></i> previous project</button>
+                    <button type="button" name="button">next project  <i class="fal fa-long-arrow-right" aria-hidden="true"></i></button>
+                  </div>
+                </div>
```

Comment:
- [ ] Kindly the project pop-up card modal suppose to be generated dynamically using javascript too like the project work cards so Please remove this code from HTML and implement it using JS.